src/sentry/scim/endpoints/teams.py

```diff
@@ -37,15 +39,6 @@
 CONFLICTING_SLUG_ERROR = "A team with this slug already exists."
```

Comment:
Unrelated nit: Seems like error messages are being kept in `constants.py`, might be good to do move this too.